src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests.cs

```diff
@@ -4879,12 +4879,14 @@ class C { }
         public void AttributeArgumentAsEnumFromMetadata()
         {
             var metadataStream1 = CSharpCompilation.Create("bar.dll",
+                options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary),
```

Comment:
>new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary) [](http://example.com/codeflow?start=25&length=65)

We have a bunch of predefined options in ```TestOptions```. consider using one of them instead.

Reply:
I intentionally didn't use those to have a minimal change to the test. The test doesn't use test helpers to create the compilation either.